scripts/UPDATE.sh

```diff
@@ -255,6 +255,10 @@ if [ -d STIR-exercises ]; then
   git pull
 fi
 
+# install the CIL-Demos
+cd $SIRF_SRC_PATH
+git clone --depth=1 --branch symposium2019  https://github.com/vais-ral/CIL-Demos.git
```

Comment:
this needs to be guarded if it exists already. best to use `clone_or_pull`
will also need the `nbstripout`

Reply:
I added a `.gitattributes` to the [`CIL-Demos` repo](https://github.com/vais-ral/CIL-Demos/blob/symposium2019/.gitattributes)